src/district_registry/server/db.cljs

```diff
@@ -25,89 +26,85 @@
    [:reg-entry/creator address not-nil]
    [:reg-entry/deposit :unsigned :BIG :INT not-nil]
    [:reg-entry/created-on :unsigned :integer not-nil]
-   [:reg-entry/challenge-period-end :unsigned :integer not-nil]
+   [:reg-entry/challenge-period-end :unsigned :integer not-nil]])
+
+(def districts-columns
+  [[:reg-entry/address address not-nil]
+   [:district/info-hash ipfs-hash not-nil]
+   [:district/dnt-weight :unsigned :integer not-nil]
+   [:district/dnt-staked :unsigned :integer not-nil]
```

Comment:
Actually thinking about this now, we should probably have table for each user's stake, because we should have option to order by amount a user staked. Can you please modify in that way?